### DISCUSSION TITLE
How to solve a max flow with min cost problem using or-tools?

### DISCUSSION THREAD

**Discussion by felipheggaliza** [google/or-tools#3451] [Graph and Algorithms] [answered]
Do someone know how I could use or-tools to solve a max flow with min cost problem?

I am trying to find something similar to the [max_flow_min_cost from NetworkX](https://networkx.org/documentation/stable/reference/algorithms/generated/networkx.algorithms.flow.max_flow_min_cost.html). 

I found [this code](https://github.com/networkx/networkx/blob/ead0e65bda59862e329f2e6f1da47919c6b07ca9/networkx/algorithms/flow/mincost.py#L327) in NetworkX repository, I am not sure yet but it seems it first solves the max flow first and later it solves the min cost problem. 

Can someone give me an example of code snippet for solving max_flow_min_cost using or-tools?

**Comment by jmarca** [comment, marked as answer]
Maybe start with the excellent documentation?

https://developers.google.com/optimization/flow/maxflow


James
